safe/gui/tools/peta_bencana_dialog.py

```diff
@@ -243,9 +243,19 @@ def accept(self):
 
         self.copy_keywords(output_base_file_path)
         layer = self.add_flooded_field(output_base_file_path)
-        # add the layer to the map
-        registry = QgsMapLayerRegistry.instance()
-        registry.addMapLayer(layer)
+
+        # check if the layer has feature or not
+        if layer.featureCount() <= 0:
+            message = 'There are no floods data available at this time.'
```

Comment:
the extent is defined by the requested url. `https://data.petabencana.id/floods?city=jbd&geoformat=geojson&format=json&minimum_state=1`, `city=jbd` will return the data only for Jabodetabek area. So, maybe it's better like this "at this time in Jabodetabek"?